class.jetpack-admin.php

```diff
@@ -95,6 +95,10 @@ function get_modules() {
 				}
 				$module_array['long_description']  = ob_get_clean();
 
+				ob_start();
+				do_action( 'jetpack_search_terms_' . $module );
```

Comment:
No reason other than consistency with how we add info for other modules.
It might be useful though.  Should we make the long_description filterable also?